tests/baselines/reference/parserGreaterThanTokenAmbiguity10.js

```diff
@@ -7,5 +7,5 @@
 //// [parserGreaterThanTokenAmbiguity10.js]
 1
     // before
-    >>>// after
+    >>> // after

```

Comment:
This comment doesn't have a leading newline, though? And the output for this test now exactly matches the input.